junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/DiscoveryFilterApplierTests.java

```diff
@@ -48,13 +45,11 @@ void packageNameFilterInclude_nonMatchingPackagesAreExcluded() {
 						classTestDescriptor("matching", MatchingClass.class)
 				)
 				.build();
-		// @formatter:on
 
 		applier.applyAllFilters(request, engineDescriptor);
 
-		List<UniqueId> includedDescriptors = engineDescriptor.getDescendants().stream().map(
-			TestDescriptor::getUniqueId).collect(Collectors.toList());
-		Assertions.assertEquals(0, includedDescriptors.size());
+		assertThat(engineDescriptor.getDescendants()).isEmpty();
+		// @formatter:on
```

Comment:
I think you can move these back up now, at least the ones for "empty".

Reply:
Hi @marcphilipp, can you clarify for me what you mean by "move these back up"? Would you like me to remove the blank line between `applier.applyAllFilters(request, engineDescriptor);` and `assertThat(engineDescriptor.getDescendants()).isEmpty();` (and so the same for all the other "empty" assertions)? Or have I misunderstood your request?

Reply:
I was referring to the line I commented on `// @formatter:on`.

Reply:
Oh, I see now. Silly me! Thanks @marcphilipp. :)